cli/log.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"fmt"
-	logpkg "log"
+	logpkg "github.com/Sirupsen/logrus"
```

Comment:
Actually it's supposed to be all lowercase, per their README: https://github.com/sirupsen/logrus/blob/master/README.md

Reply:
@swoirgard  Done